Loop tests fail sometimes

The test `TestExecuteInterruptDoesNotCancelLoop` is failing sometimes on travis.

```
=== RUN   TestExecuteSubShellDoesNotOverwriteparentEnv
--- PASS: TestExecuteSubShellDoesNotOverwriteparentEnv (0.00s)
=== RUN   TestExecuteInterruptDoesNotCancelLoop
--- FAIL: TestExecuteInterruptDoesNotCancelLoop (0.00s)
	shell_test.go:1644: loop interrupted
=== RUN   TestExecuteBuiltin
--- PASS: TestExecuteBuiltin (0.03s)
```

--- a/internal/sh/shell_test.go
+++ b/internal/sh/shell_test.go
@@ -1679,6 +1679,8 @@ func TestExecuteInterruptDoesNotCancelLoop(t *testing.T) {
 
 	sh.TriggerCTRLC()
 
+	time.Sleep(time.Second * 1)
+
 	err = sh.Exec("interrupting loop", `seq = (1 2 3 4 5)
 for i in $seq {}`)
 

